Blockzilla/Modules/WebView/WebViewController.swift

```diff
@@ -150,6 +152,8 @@ class WebViewController: UIViewController, WebController {
         browserView.snp.makeConstraints { make in
             make.edges.equalTo(view.snp.edges)
         }
+        
+        KVOs.forEach { browserView.addObserver(self, forKeyPath: $0.rawValue, options: .new, context: nil) }
```

Comment:
Instead of creating this variable and using it here can we use `KVOKeyPath.allCases` yet?

Reply:
<img width="781" alt="image" src="https://user-images.githubusercontent.com/1250545/47927872-bb572c80-de81-11e8-93d8-95f341ba825e.png">